docs/CaliptraIntegrationSpecification.md

```diff
@@ -240,13 +240,13 @@ SoC must ensure that there are no SCAN cells on the flops that latch this key in
 
 The interface signals GENERIC\_INPUT\_WIRES and GENERIC\_OUTPUT\_WIRES are placeholders on the SoC interface reserved for late binding features. This may include any feature that is required for correct operation of the design in the final integrated SoC and that may not be accommodated through existing interface signaling (such as the mailbox).
 
-While these late binding interface pins are generic in nature until assigned a function, integrators must not define non-standard use cases for these pins. Defining standard use cases ensures that the security posture of Caliptra in the final implementation is not degraded relative to the consortium design intent.
+While these late binding interface pins are generic in nature until assigned a function, integrators must not define non-standard use cases for these pins. Defining standard use cases ensures that the security posture of Caliptra in the final implementation is not degraded relative to the consortium design intent. Any bits in GENERIC\_INPUT\_WIRES for which a function has not been defined (in the Caliptra version that is being integrated) must be tied to a 0-value. These undefined input bits shall not be connected to any flip flops (which would allow run time transitions on the value).
```

Comment:
Could "in the Caliptra version that is being integrated" be "in Caliptra" because it's a given that we're referring to the version that is being integrated? 

- If so, could simplify to: "Bits in GENERIC\_INPUT\_WIRES that don't have a function defined in Caliptra must be tied to a 0-value." 

- If no, then maybe these minor mods: "Bits in GENERIC\_INPUT\_WIRES that don't have a function defined (in the Caliptra version that is being integrated) must be tied to a 0-value." 

- also "run time" should be "run-time" in this instance (it is modifying "transitions")

Reply:
My goal with this language is to differentiate between the latest Caliptra code in the tip of the repository, vs the integration version that was consumed (e.g, new bits might be added in a hypothetical 1.2 version or 2.0 version of Caliptra that should be tied to 0 by integrators who are consuming 1.1). 
If this comes across as implicitly obvious, I can remove the parentheses.

Reply:
Thanks for the explanation, Caleb! We don't really refer to "version" explicitly in other areas of the doc, so my assumption is we are documenting the current supported version unless explicitly stated otherwise. I'd defer to you and other reviewers, though, if the differentiation is needed here.

Reply:
Great recommendations, I have removed the version bit :)